third_party/txt/src/txt/font_collection.cc

```diff
@@ -170,6 +165,12 @@ FontCollection::GetMinikinFontCollectionForFamilies(
       }
     }
   }
+  // Default font family and fallbacks also not found. We fail to get a
```

Comment:
If the engine reached this point, then it was unable to find any of the specified font families as well as the platform default font family (which should always be present). 
 When is this happening?

The fallback fonts are going to be an arbitrary collection of fonts that happened to be discovered in past lookups.  I'd be concerned that using them will lead to inconsistent behavior.

Reply:
I'm not certain this actually happens in the wild, but I got it while debugging  https://github.com/flutter/flutter/issues/41101 in a very specific artificially supplied edge case.

I guess the question is if rendering something possibly inconsistent is better than rendering nothing. I suspect it wont matter either way as this is highly unlikely if not impossible to occur.